docs/data-management/data-pipeline.md

```diff
@@ -10,7 +14,27 @@ At a high level, the Eppo experiment data pipeline has four main branches along
 
 ![Experiment data pipeline diagram](/img/experiments/data-pipeline/experiment_pipeline_visualization.png)
 
-## Assignment summarization
+### Monitoring pipeline performance
+
+To see how compute resources are used across the different sections of the experiment pipeline, navigate to the warehouse tab on the left navbar. Here you can see a recent trend of pipeline runtime by section:
+
+![Data pipeline chart](/img/data-management/pipeline/warehouse-by-task.png)
+
+Note that the y axis shows the total time accrued by that task type. That is, if two queries run at the same time for 5 minutes, that would contribute 10 minutes to the y axis.
+
+### Incremental refreshes
+
+Eppo's nightly job will run an incremental refresh that only scans recent data. By default, this lookback window will include data starting 48 hours before the last successful run (to change this time window, reach out to your Eppo contact or email support@geteppo.com). 
```

Comment:
Not all customers run it “nightly”: “scheduled” or “daily (usually overnight)” might confuse people less.

Reply:
> only scans recent data
"and facts needed for metrics recently added to the analysis"? That’s a common source of confusion. Not sure how to phrase it in a way that isn’t too cumbersome.